### PR TITLE
Remove trailing zeros in cdc testing

### PR DESCRIPTION
**Pull request by ghop02**
First attempt at removing trailing zeros

### CODE REVIEW

File: scripts/update_cdc_test_data.py [hunk 1]
```diff
@@ -34,6 +34,31 @@ class Fields(GetByValueMixin, FieldNameAndCommonField, enum.Enum):
     VALUE = "value", None
 
 
+def _remove_trailing_zeros(series: pd.Series) -> pd.Series:
+
+    series = series.copy()
+
+    index = series.loc[series != 0].last_valid_index()
```
Comment (TomGoBravo) [inline scripts/update_cdc_test_data.py:41]: What happens for NA values?

File: scripts/update_cdc_test_data.py [hunk 2]
```diff
@@ -34,6 +34,31 @@ class Fields(GetByValueMixin, FieldNameAndCommonField, enum.Enum):
...
+
+    if index is None:
+        # If test positivity is 0% the entire time, considering the data inaccurate, returning
+        # none.
+        series[:] = None
+        return series
+
+    series[index + pd.DateOffset(1) :] = None
```
Comment (TomGoBravo) [inline scripts/update_cdc_test_data.py:49]: This DateOffset(1) thing is kind of ugly. Why not return `series[:index]` ?

**Review by TomGoBravo — APPROVED** [review]
Needs some tests but otherwise looks like it'll work.

File: scripts/update_cdc_test_data.py (hunk 2 above)
Comment (TomGoBravo) [reply scripts/update_cdc_test_data.py:49]: Answering my own question, I suspect that `data[CommonFields.TEST_POSITIVITY_7D] = test_pos` only sets values in data with index returned by groupby, leaving zeros after `series[:index]` in place.

File: scripts/update_cdc_test_data.py (hunk 1 above)
Comment (ghop02) [reply scripts/update_cdc_test_data.py:41]: hmm fwiw I don't think there are any NA values

**Pull request merged**